.github/renovate.json

```diff
@@ -25,12 +25,46 @@
       "enabled": false
     },
     {
-      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204", "custom.deb2404"],
+      "matchDatasources": ["docker"],
```

Comment:
Splitting the original single block into different because of a branch collision issue.
Before this change, updating containerd for Ubuntu 20.04 and 22.04 will use the same branch name. And the latter one will run into branch collision issue. That's why we only saw auto-update PR for 20.04 but not 22.04. Here we add `additionalBranchPrefix` for each ubuntu release so the branch name will become renovate/[custom datasource name]-moby-containerd-1.7.x to avoid collision. Tested in my fork. Now we can see both 20.04 and 22.04 PRs will be created.

Reply:
![image](https://github.com/user-attachments/assets/afb2f037-0e3c-4cfb-93c2-609e6d2f4652)